### PR TITLE
OSDOCS#8972: Release note for disabling DC capability not generating …

### PR DESCRIPTION
**Pull request by bergerhoffer**
…deployer SA

Version(s):
4.15

Issue:
https://issues.redhat.com/browse/OSDOCS-8972

Link to docs preview:
https://71447--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-auth-deployer-sa

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@ardaguclu @kasturinarra ready for review please!

Preview: https://71447--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-auth-deployer-sa

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -764,6 +764,13 @@ This release introduces a configuration adjustment in response to the transition
 
 To resolve these warnings, we recommend specifying `-1` or `auto` for the `maxConnections` field when tuning an IngressController. This choice allows HAProxy to dynamically calculate the maximum value based on the available resource limitations in the running container, which eliminates these warnings. (https://issues.redhat.com/browse/OCPBUGS-21803[*OCPBUGS-21803*])
 
+[id="ocp-4-15-auth-deployer-sa"]
+==== The deployer service account is no longer created if the DeploymentConfig cluster capability is disabled
+
+If you disable the `DeploymentConfig` cluster capability, the `deployer` service account its corresponding secrets are no longer created.
```
Comment (agantony) [inline release_notes/ocp-4-15-release-notes.adoc:770]:
```suggestion
If you disable the `DeploymentConfig` cluster capability, the corresponding secrets of the `deployer` service account are no longer created.
```

Comment (agantony) [reply release_notes/ocp-4-15-release-notes.adoc:770]: Hope the technical meaning does not change?

**Review by agantony — COMMENTED** [review]
Added a small suggestion for your consideration; overall lgtm!

File: release_notes/ocp-4-15-release-notes.adoc (hunk 1 above)
Comment (bergerhoffer) [reply release_notes/ocp-4-15-release-notes.adoc:771]: It does, but I can see why you changed it - I'm missing an "and" in there. Will update, thanks!

**Pull request merged**